src/collapse/perf.rs

```diff
@@ -519,8 +519,8 @@ impl Folder {
                 } else if self.opt.annotate_kernel && is_kernel(module) {
                     func.push_str("_[k]"); // kernel
                 } else if self.opt.annotate_jit
-                    && module.starts_with("/tmp/perf-")
-                    && module.ends_with(".map")
+                    && ((module.starts_with("/tmp/perf-") && module.ends_with(".map"))
+                        || (module.contains("/jitted-") && module.ends_with(".so")))
```

Comment:
Should this be `starts_with`, not `contains`?

Reply:
`perf inject` will create the ELF files wherever the jitdump metadata file was located, and unlike the old jit interface it can be anywhere on the filesystem, so `contains` is correct here.